### ISSUE TITLE
Install instructions

### ISSUE THREAD

**Issue by devnoname120** [dnamsons/obsidian-publish-gist#7]
Hi! Thank you for this plugin. I was looking exactly for this!

I compiled the project and it produces the `main.js` file. How to install it in Obsidian?

**Comment by dnamsons** [comment]
@devnoname120 Hey! Glad you find this useful! 

You can just download the zip from the releases page(https://github.com/dnamsons/obsidian-publish-gist/releases/tag/1.0.1), extract it and move the extracted folder into your vault's `/.obsidian/plugins` directory

If you want to do this manually, you can create a new folder, move the compiled `main.js`, `manifest.json` and the `README.md` into it(It might work without the README, but I am not 100% sure about that) and move the folder into your vault's `/.obsidian/plugins` directory

I will eventually publish this plugin to the official plugin list, but I want to make this plugin more feature complete before i do so.

**Comment by dnamsons** [comment]
@devnoname120 Yeah, you're right. I have now added a Github actions workflow that will automatically build a zip file of the necessary files - https://github.com/dnamsons/obsidian-publish-gist/releases/tag/1.1.0 already contains the zip file with the contents(`obsidian-publish-gist-1.1.0.zip`)
